Add a Print Notations command (#15683)
coq/coq: doc/changelog/07-vernac-commands-and-options/14907-14907-print-notation.rst

- **Added:**

>   Added a :cmd:`Print Notation` command which when given a notation
>   string, as used when declaring a notation, prints the level and
>   associativity.

Reviewer: ```suggestion
  :cmd:`Print Notation` command that prints the level and
  associativity of a given notation definition string
```

No period before the `(` on line 5.
